rules/windows/process_creation/proc_creation_win_unusual_svchost_command.yml

```diff
@@ -0,0 +1,31 @@
+title: Unusual Svchost Command Line Parameter
+id: f17211f1-1f24-4d0c-829f-31e28dc93cdd
+status: experimental
+description: |
+    This rule alerts when svchost.exe is executed with unusual command line parameters based on known good patterns.
+    This could point at a file masquerading as svchost or a process injection or hollowing of a legitimate svchost instance.
+references:
+    - https://cardinalops.com/blog/the-art-of-anomaly-hunting-patterns-detection/
+    - https://www.security.com/threat-intelligence/blackbyte-exbyte-ransomware
+    - https://cloud.google.com/blog/topics/threat-intelligence/apt41-initiates-global-intrusion-campaign-using-multiple-exploits/
+    - https://media.kasperskycontenthub.com/wp-content/uploads/sites/43/2018/03/08064518/Carbanak_APT_eng.pdf
+author: Liran Ravich
+date: 2025-07-28
+tags:
+    - attack.defense-evasion
+    - attack.privilege-escalation
+    - attack.t1036.005
+    - attack.t1055
+    - attack.t1055.012
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\svchost.exe'
+    filter_cmd:        # Example of command to simulate: "c:\Windows\System32\svchost.exe" calc.exe
+        CommandLine|re: '.*-k\s\w+((\s?(-p|-s)).*|$)'
```

Comment:
```suggestion
        CommandLine|re: '-k\s\w{1,64}(\s?(-p|-s)|$)'
```

Reply:
`|re` matches as `contains`. So surrounding `.*` are redundant. In general we do not allow unbound wildcard matching.